app/helpers/course/users_helper.rb

```diff
@@ -1,4 +1,24 @@
 # frozen_string_literal: true
 module Course::UsersHelper
-  include Course::Achievement::AchievementsHelper
+  # Returns a hash that maps +User+ ids to their +CourseUser+ IDs in a given +course_id+
+  #
+  # @param [Integer] course_id The ID of the course
+  # @return [Hash]
+  def preload_course_user_ids_hash(course_id)
```

Comment:
This function is not used anywhere right?

Reply:
Not that I know of. I left it there in case someone needs it, but I guess when that time comes, they can write the same function here again. I will remove it.